OpenRA.Mods.Common/Widgets/Logic/ConnectionLogic.cs

```diff
@@ -167,7 +189,7 @@ public ConnectionSwitchModLogic(Widget widget, OrderManager orderManager, Networ
 				var launchCommand = $"Launch.URI={new UriBuilder("tcp", connection.EndPoint.Address.ToString(), connection.EndPoint.Port)}";
 				Game.SwitchToExternalMod(CurrentServerSettings.ServerExternalMod, new[] { launchCommand }, () =>
 				{
-					orderManager.ServerError = "Failed to switch mod.";
+					orderManager.ServerError = modData.Translation.GetString(ModSwitchFailed);
```

Comment:
`orderManager.ServerError` is run through `modData.Translation.GetString`, so not needed here (actually, the real fix will be to implement proper translation-aware plumbing for this, but that can its own PR).